.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+    push:
+        branches:
+            - master
+jobs:
+    release:
+        name: Build and Release
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+            issues: write
+            pull-requests: write
+            id-token: write
+        steps:
+            - name: Checkout
+              uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+              with:
+                  fetch-depth: 0
+            - name: Setup Node.js
+              uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8
+              with:
+                  node-version: 20.x
+            - name: Install Dependencies
```

Comment:
Can we replace this step with the one in the shared action 🤔 
https://github.com/niloofar-deriv/shared-actions/tree/master/.github/actions

Reply:
Alright, I'll replace it with that.